json/src/main/java/com/basistech/rosette/dm/jackson/AnnotatedDataModelModule.java

```diff
@@ -89,6 +90,7 @@ public void setupModule(SetupContext context) {
         context.setMixInAnnotations(com.basistech.rosette.dm.ResolvedEntity.class, ResolvedEntityMixin.class);
         context.setMixInAnnotations(Sentence.class, SentenceMixin.class);
         context.setMixInAnnotations(ScriptRegion.class, ScriptRegionMixin.class);
+        context.setMixInAnnotations(LayoutRegion.class, LayoutRegionMixin.class);
```

Comment:
This entry, and similar ones in similar lists, should be added in alphabetical order.